install/cloudflare-ddns-install.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: edoardop13
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://github.com/favonia/cloudflare-ddns
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing dependencies"
+$STD apt-get update
+$STD apt-get install -y curl systemd
```

Comment:
```suggestion
msg_info "Installing dependencies"
$STD apt-get install -y systemd
msg_ok "Installed dependencies"
```

Update is done by the update_os function above